frame/assets/src/lib.rs

```diff
@@ -49,8 +49,18 @@
 //! - `mint_into`
 //! - `burn_from`
 
-#![cfg_attr(not(test), warn(clippy::disallowed_method, clippy::indexing_slicing))] // allow in tests
-#![warn(clippy::unseparated_literal_suffix, clippy::disallowed_type)]
+#![cfg_attr(
+	not(test),
+	warn(
+		clippy::disallowed_method,
+		clippy::disallowed_type,
+		clippy::indexing_slicing,
+		clippy::todo,
```

Comment:
how we should mark to do items? todo was nice way to do it

Reply:
I would say that we shouldn't mark todo items, and prs should either finish the implementation or not be merged

Reply:
I can remove the todo macro from the crates, and enable it in CI for releases if that makes more sense. But I think for PR's to main, `todo` should be handled by then to be honest.